mobile/src/services/notification/notification.service.ts

```diff
@@ -0,0 +1,30 @@
+import Toast from 'react-native-toast-message';
+import { NotificationType } from '~/common/enums/enums';
+
+class Notification {
+  private [NotificationType.ERROR](message: string): void {
```

Comment:
all these methods should be public, you need to create only one private method like showNotification and call Toast.show only from it. In other methods you will just call showNotification and pass needed config to it

Reply:
Ohh, I didn't think about it, something like that?
![image](https://user-images.githubusercontent.com/82529236/183574325-85f245d5-45ac-4b16-a90a-ddabd4aa9af3.png)